modules/database/src/ioc/db/dbIocRegister.c

```diff
@@ -235,14 +235,16 @@ static void dblaCallFunc(const iocshArgBuf *args) { iocshSetError(dbla(args[0].s
 
 /* dbgrep */
 static const iocshArg dbgrepArg0 = { "pattern",iocshArgStringRecord};
-static const iocshArg * const dbgrepArgs[1] = {&dbgrepArg0};
-static const iocshFuncDef dbgrepFuncDef = {"dbgrep",1,dbgrepArgs,
-                                           "List record names matching pattern.\n"
+static const iocshArg dbgrepArg1 = { "field",iocshArgStringRecord};
```

Comment:
I don't think this argument should be `iocshArgStringRecord` since that tells the iocsh completion code to try to complete it as a record name. It should just be `iocshArgString`.

Reply:
Ah yes, careless cut and paste on my part - thanks